### DISCUSSION TITLE
Unable to use SQLCipher

### DISCUSSION THREAD

**Discussion by oddanderson** [groue/GRDB.swift#1141] [Q&A] [answered]
Hi there - I'm running into what seems like a common issue:

Fatal error: Unresolved error SQLite error 21: GRDB is not linked against SQLCipher. Check https://discuss.zetetic.net/t/important-advisory-sqlcipher-with-xcode-8-and-new-sdks/1688

I have followed the instructions in that article, such that my `-framework "SQLCipher"` is listed before `-framework "GRDB"`
in the following places:
Project settings - Other Linker Flags
Build Target settings - Other Linker Flags.

Running Clean -> Build results in the same issue. Am I missing something here?

my podfile:
```
pod 'GRDB.swift'
  pod 'GRDB.swift/SQLCipher'
  pod 'SQLCipher', '~> 3.0'
```

Linker flags
```
//:configuration = Debug
OTHER_LDFLAGS = $(inherited) -ObjC -l"c++" -framework "SQLCipher" -l"sqlite3" -l"z"<...> -framework "GRDB"<...>

//:completeSettings = none
```

**Comment by groue** [comment, marked as answer]
Hello @oddanderson,

I'm surprised by `-l"sqlite3"`, since you _don't want_ to link against the regular SQLite.

**Reply by groue** [reply]
Also, if you use `pod 'GRDB.swift/SQLCipher'`, then remove `pod 'GRDB.swift'`.

**Reply by groue** [reply]
Generally speaking, if the standard [instructions](https://github.com/groue/GRDB.swift/blob/master/README.md#encryption) can fit your needs, then following them is a good way to avoid problems (`'~> 3.0'` 👀)

**Reply by oddanderson** [reply]
That's coming from Firebase. Removing the first pod declaration resolved the issue. FWIW I had no intention of deviating from the standard instructions, I just didn't realize I needed to remove the original pod reference

**Reply by groue** [reply]
> I just didn't realize I needed to remove the original pod reference

OK. I'll update the doc to make this clear.

EDIT: https://github.com/groue/GRDB.swift/commit/8f100a63950d2097d875826d5892b7e5c75a7552

Thank you!

**Reply by oddanderson** [reply]
Thanks! Great library btw - much preferred over Core Data

**Reply by groue** [reply]
Happy GRDB :-)
